src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,52 @@
+package parser;
+
+import org.junit.jupiter.api.*;
+import shop.Cart;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JsonParserTest {
+    static File file;
+    @BeforeAll
+    static void fileCreation(){
```

Comment:
please add 1 empty line between lines of code

Reply:
Done